app/models/workflow_execution.rb

```diff
@@ -87,4 +87,27 @@ def validate_namespace
 
     errors.add(:base, 'workflow executions can only belong to a Project or Group namespace')
   end
+
+  private
+
+  def send_user_emails
+    if completed?
+      PipelineMailer.complete_user_email(self).deliver_later
+    elsif error?
+      PipelineMailer.error_user_email(self).deliver_later
+    end
+  end
+
+  def send_manager_emails
+    I18n.available_locales.each do |locale|
+      manager_emails = Member.manager_emails(namespace, locale)
```

Comment:
This only gets emails for the managers that are members of the namespace. However if the namespace is a user namespace it doesn't get the email for the user who is the owner of the project

Reply:
You can get the owner of the namespace if the namespace parent is a user namespace. Then if the users locale matches the current locale iteration, you can add the owners email (using `<<`) to the array returned for manager_emails

Reply:
c29db644d29ab5fc79fe1f6ba54a8e1c25496a79